js/main.js

```diff
@@ -0,0 +1,19 @@
+const getRandomNumber = function(min, max) {
+  if (min >= max || min < 0 || max < 0)
+    return false
+  return Math.floor(Math.random() * (max - min + 1)) + min; //Максимум и минимум включаются
+}
+// from
+// https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+
+const getRandomNumberComma = function(min, max, comma) {if (min >= max || min < 0 || max < 0)
```

Comment:
```suggestion
const getRandomNumberComma = (min, max, comma) => {if (min >= max || min < 0 || max < 0)
```
Так же как выше, давай использовать стрелочную функцию здесь

Не понятно почему `if` идет в одной строку тут. Нужно исправить. И не забудь про кавычки для блока `if`